src/appforms/src/backbone/040-view06Steps.js

```diff
@@ -71,12 +71,14 @@ StepsView = Backbone.View.extend({
 
     self.$el.find('li:eq(' + displayIndex + ')').addClass('active');
 
-    if(pageName.length === 0){
-      pageName = "Page " + (displayIndex + 1);
+    var hasCustomPageName = pageName.length !== 0;
```

Comment:
@nialldonnellyfh Would you mind taking another look. I've updated this PR with an additional commit.